src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -250,7 +251,7 @@ public void actionPerformed(ActionEvent e) {
         @Override
         public void actionPerformed(ActionEvent e) {
             if (e.getSource() == chkTrackingConsent) {
-                BriefcasePreferences.setBriefcaseTrackingConsentProperty(chkTrackingConsent.isSelected());
+                mainBriefcaseWindow.briefcaseAnalytics.trackConsentDecision(chkTrackingConsent.isSelected());
```

Comment:
I missed it, my comment is technically correct but yours is more straight to the point, I'll amend mine.